drakcore/drakcore/frontend/src/AnalysisStatus.js

```diff
@@ -28,19 +83,58 @@ class AnalysisStatus extends Component {
         let newSpinner = this.state.spinner === "oO" ? "Oo" : "oO";
         this.setState({
           status: response.data.status,
+          vm_id: parseInt(response.data.vm_id),
           updated: true,
           spinner: newSpinner,
         });
       }
     }, 1000);
   }
 
+  createConnection() {
+    let rfb = null;
+    try {
+      rfb = new RFB(
+        document.getElementById("noVNC-canvas"),
```

Comment:
If I'm not mistaken we should use React ref instead of div IDs

Reply:
yes